app/regen/cmd/root.go

```diff
@@ -69,26 +66,6 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 	return rootCmd, encodingConfig
 }
 
-// Execute executes the root command.
-func Execute(rootCmd *cobra.Command) error {
```

Comment:
Why is this deleted?

Reply:
In Cosmos SDK simapp we're now using sdk's server.Execute(), i figured this was what we should update to as well. There's no need to have this function defined in regen-ledger if its exposed on the SDK right?

Reply:
@anilCSE Hmm... Yeah you're 100% right here. Seems strange then that `server.Execute()` is exposed at all if it doesn't provide a mechanism for chains to add their own prefix... Do you think this should be addressed upstream?

Maybe best to revert this part of my PR for now.

Reply:
Yes, we can fix this upstream to take an extra argument for this ENV prefix.

Reply:
Opened a pull request in the sdk: https://github.com/cosmos/cosmos-sdk/pull/10950